packages/proto-build/src/main.rs

```diff
@@ -5,30 +5,12 @@
 
 use std::{env, path::PathBuf};
 
+use dotenvy;
```

Comment:
It looks `dotenvy` cannot be imported. Declare to use it in Cargo.toml.

Reply:
I forgot to commit. thank you.